### ISSUE TITLE
Cannot find the Physio Recordings in VA30A SP03

### ISSUE THREAD

**Issue by WMallon** [CMRR-C2P/MB#369]
System: Siemens Prisma  NX VA30A SP03
CMRR Version: CMRR_MB_EPI_XA30A_R017pre5

Even though the Physio recordings have been enabled in the legacy mode (DICOM mode is not available, why?) no log files have been found.

These were the folders we looked into:
C:\MedCom\Log\Physio (as reported in the installation instructions)
C:\ProgramData\Siemens\Numaris\log
C:\Program Files\Siemens\Numaris\Mars\MriCustomer\Log\Physio
C:\Program Files\Siemens\Numaris\Mars\MriCustomer\CustomerSeq\Log\Physio

I appreciate any feedback.

WM

**Comment by eauerbach** [comment]
Yes, if you update to the latest version the DICOM mode will be available. Otherwise the correct output path for the log files should be shown in the tooltip; that might not match what is in the documentation (which may not have been updated for XA).

**Comment by eauerbach** [comment]
...and yes you can just rerun the installer without installing again the certificate, and the data will be compatible
